### ISSUE TITLE
failed to jump in normal mode

### ISSUE THREAD

**Issue by redguardtoo** [redguardtoo/evil-matchit#43]
move the cursor to the beginning of first line, does not work:
```html
<div ng-if="!viewBeneficiariesController.isNotAllowedToListBeneficiaries">
    <section class="alert alert--info" ng-show="viewBeneficiariesController.isNoBeneficiaries">
        <p class="alert__message">You do not have any beneficiaries currently nominated</p>
    </section>
    <div ng-if="!viewBeneficiariesController.liftAndShift" ng-switch on="viewBeneficiariesController.beneficiaryDetails.type">
        <div ng-switch-when="NON_BINDING">
            <div ng-include="'beneficiary-details/view.beneficiaries--non-binding.html'"></div>
        </div>
        <div ng-switch-when="BINDING">
            <div ng-include="'beneficiary-details/view.beneficiaries--binding.html'"></div>
        </div>
        <div ng-switch-when="REVERSIONARY">
            <div ng-include="'beneficiary-details/view.beneficiaries--reversionary.html'"></div>
        </div>
        <div ng-switch-when="NOMINATED_NON_BINDING">
            <div ng-include="'beneficiary-details/view.beneficiaries--nominated-non-binding.html'"></div>
        </div>
        <div ng-switch-when="NOMINATED_BINDING">
            <div ng-include="'beneficiary-details/view.beneficiaries--nominated-binding.html'"></div>
        </div>
        <div ng-switch-when="NOMINATED_REVERSIONARY">
            <div ng-include="'beneficiary-details/view.beneficiaries--nominated-reversionary.html'"></div>
        </div>
        <div ng-switch-default>
            <button class="btn btn--primary btn--fluid btn--alpha mt mb" disabled>+ Add non-binding beneficiary</button>
        </div>
    </div>
    <div ng-if="viewBeneficiariesController.liftAndShift" ng-include="'beneficiary-details/view.beneficiaries--lift-and-shift.html'"></div>
</div>
<div ng-if="viewBeneficiariesController.isNotAllowedToListBeneficiaries">
    <button class="btn btn--primary btn--fluid btn--alpha mt mb" disabled>+ Add non-binding beneficiary</button>
</div>
```

**Comment by redguardtoo** [comment]
4f92c90 #43 cannot jump at the first line in html (Chen Bin)

**Comment by redguardtoo** [comment]
3f08492 FIXED: #43 no jump at the first line in html v1.5.2 (Chen Bin)
